py/selenium/webdriver/ie/service.py

```diff
@@ -40,16 +39,20 @@ def __init__(
          - executable_path : Path to the IEDriver
          - port : Port the service is running on
          - host : IP address the service port is bound
-         - log_level : Level of logging of service, may be "FATAL", "ERROR", "WARN", "INFO", "DEBUG", "TRACE".
-           Default is "FATAL".
+         - log_level : Level of logging of service, may be "DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL".
+           Default is "WARNING".
```

Comment:
default is determined by driver

Reply:
ohh sorry.. kind of confused with regular python log levels..